fairlearn/datasets/_fetch_compas.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import pathlib
+import warnings
+
+from sklearn.datasets import fetch_openml
+from ._constants import _DOWNLOAD_DIRECTORY_NAME
+
+from fairlearn.exceptions import DataFairnessWarning
+
+
+def fetch_compas(*, cache=True, data_home=None,
```

Comment:
As per the conversation elsewhere, let's change this:
* fetch_compas -> fetch_compas_propublica
(and similarly rename the file)